pkg/kubelet/cm/topologymanager/fake_topology_manager.go

```diff
@@ -27,31 +27,31 @@ type fakeManager struct{}
 
 //NewFakeManager returns an instance of FakeManager
 func NewFakeManager() Manager {
-	klog.Infof("[fake topologymanager] NewFakeManager")
+	klog.InfoS("[fake topologymanager] NewFakeManager")
```

Comment:
```suggestion
	klog.InfoS("NewFakeManager")
```

Please remove log prefixes